### DISCUSSION TITLE
blog/pagination-in-aspnet-core-webapi/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#22] [General]
# blog/pagination-in-aspnet-core-webapi/

In this guide, we will learn how to implement Advanced Pagination in ASP.NET Core WebApi with ease. Pagination is one of the most important concepts while building RESTful APIs. You would have seen several public APIs implementing this feature for better user experience and security. We will go in detail and try to build an ASP.NET Core 3.1 WebApi that implements advanced pagination.

https://codewithmukesh.com/blog/pagination-in-aspnet-core-webapi/

**Comment by phamquangvinhfpt** [comment]
How about searching, filtering and sorting sir?
